app.js

```diff
@@ -204,9 +203,6 @@ if (useDocumentation || onlyDocumentation === 'true') {
     extended: true,
   }));
 
-  // Custom documentation routes
-  documentationApp.use('/', documentationRoutes);
-
   // Automatically route documentation pages
```

Comment:
Delete?

Reply:
Wasn't sure if this bit was still needed?